.github/workflows/build-components.yml

```diff
@@ -4,7 +4,6 @@ on:
   pull_request:
       branches:
         - main
-        - gcds-components-ssr
```

Comment:
Do we want to keep this for now so we can check the build status on PRs against the SSR branch, if we're releasing form components first?

Reply:
Good idea